src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2482,9 +2487,25 @@ private static byte[] StringToUTF8(string s)
 
         private byte* getMethodNameFromMetadata(CORINFO_METHOD_STRUCT_* ftn, byte** className, byte** namespaceName)
         {
-            // TODO: Implement JIT recognized intrinsics
-            // https://github.com/dotnet/corert/issues/4492
-            return null;
+            MethodDesc method = HandleToObject(ftn);
+
+            MetadataType owningType = method.OwningType as MetadataType;
```

Comment:
Do we need `.GetTypeDefinition()` here?

Reply:
You mean to handle `InstantiatedType`? That one just forwards to the definition.

Reply:
Ok ... you are right.